src/panels/config/entities/ha-config-entities.ts

```diff
@@ -156,18 +167,38 @@ export class HaConfigEntities extends SubscribeMixin(LitElement) {
   private _filteredEntities = memoize(
     (
       entities: EntityRegistryEntry[],
+      states: HassEntities,
       showDisabled: boolean,
-      showUnavailable: boolean
+      showUnavailable: boolean,
+      showReadOnly: boolean
     ) => {
+      let stateEntities: EntityRegistryEntry[] = [];
+      if (showReadOnly) {
+        const regEntityIds = entities.map((entity) => entity.entity_id);
+        const stateEntityIds = Object.keys(states).filter(
+          (entityId) => !regEntityIds.includes(entityId)
+        );
+        stateEntities = stateEntityIds.map((entityId) => {
+          return {
+            name: computeStateName(states[entityId]),
+            entity_id: entityId,
+            platform: computeDomain(entityId),
+            disabled_by: null,
+            readonly: true,
+            selectable: false,
```

Comment:
I feel like it should still be selectable, but the entity registry dialog should have a message what's up instead of controls on the first page. The related tab and jumping to the controls dialog is still useful.

Reply:
Oh, I see now that selectable and open-dialog are different options.